src/main/java/frc/utils/controllers/keyboard/KeyboardController.java

```diff
@@ -0,0 +1,150 @@
+package frc.utils.controllers.keyboard;
+
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
+import frc.utils.CMDHandler;
+import frc.utils.dashboard.LoggedTableBoolean;
+
+import java.nio.file.Path;
+
+
+/*
+ * inspired from Trigon code
+ *
+ * @author Yoni Kiriaty, Trigon
+ *
+ */
+public class KeyboardController {
+
+    private static final double KEY_PRESSED_VALUE = 0.5;
+
+    private static final Path KEYBOARD_TO_NETWORK_TABLES_CLASS = Path.of("KeyboardToNetworkTables");
+
+    private static final String KEYBOARD_TABLE = "Keyboard";
+    private static final String KEYS_TAB = "Keys/";
+
+    public final Trigger
+            ESC,
+            F1, F2, F3, F4, F5, F6, F7, F8, F9, F10, F11, F12,
+            DELETE, BACKTICK,
+            ONE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE, ZERO,
+            MINUS, EQUALS,
+            BACKSPACE, TAB,
```

Comment:
kinda weird that you went down a line here but then in keyoard controller you put them togehter